src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useEffect, useMemo, useState } from 'react';
```

Comment:
When a group is selected from recommended and then deselected again the EPEL repo stays selected. It should get deselected when the last item from it is unchecked as for packages.

Also the repo then gets stuck in the list until refresh

![unselectable-repo](https://github.com/osbuild/image-builder-frontend/assets/49452678/783eaebf-f7f8-4395-9262-cc14988d69ab)